htmd/builder/builder.py

```diff
@@ -155,6 +157,15 @@ def embed(mol1, mol2, gap=1.3):
     return mol2
 
 
+def convertDisulfide(mol, disu):
+    from htmd.molecule.molecule import UniqueResidueID
+    newdisu = []
+    for d in disu:
+        if not isinstance(d[0], str) or not isinstance(d[1], str):
+            raise RuntimeError('All disulfide selections should be strings')
+        newdisu.append([UniqueResidueID.fromMolecule(mol, d[0]), UniqueResidueID.fromMolecule(mol, d[1])])
```

Comment:
I guess if there would be a cystein residue with `altloc`'ed sulphurs, it would immediately fail here (actually, in one of the fail checks inside `UniqueResidueID`) due to returning more than one atom. That's sufficient validation, correct?

Reply:
Altlocs are already thrown out by Molecule during reading. There is an option to keep them but then it's up to the user to remove/keep altlocs he wants.

Reply:
I see. Great!